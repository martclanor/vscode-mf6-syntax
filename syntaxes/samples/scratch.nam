# Scratch test data

# dfdf # sdfdf
BEGIN MODELS
  gwf6 sdfdsf#modeltype      namefile      modelname
  gwf6 sdfdsf #modeltype      namefile      modelname
         GWF6     model.nam          GWF_1
END MODELS

  BEGIN     EXCHAngES
  BEGIN     EXCHAngES.
  BEGIN     .EXCHAngES
  BEGIN.     EXCHAngES
  .BEGIN     EXCHAngES
END EXCHANGES

BEGIN
  BEGIN
  .BEGIN.
  BEGIN.
  .BEGIN
    dfdf CONSTANT #dfdf  dfdfdf
    dfdf .CONSTANT
     ...OPEN/CLOSE
     OPEN/CLOSE
     internal   factor #df
     Internal  factor#dfdf
        .gwf6-gwf6.
        gwf6-gwt6
        gwf6 - gwf6

    (BINARY)   12323
    .(BINARY)   12323
    (BINARY).   12323

alh
.alh.
alh.
.alh


drop
.drop.
drop.
drop.

BEGIN time
METHODS linear linear
IMS6
EMS6
