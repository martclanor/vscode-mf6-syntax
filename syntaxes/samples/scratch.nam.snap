># Scratch test data
#^^^^^^^^^^^^^^^^^^^ source.mf6 comment.mf6
>
># dfdf # sdfdf
#^^^^^^^^^^^^^^ source.mf6 comment.mf6
>BEGIN MODELS
#^^^^^ source.mf6 keyword.control.mf6
#     ^ source.mf6
#      ^^^^^^ source.mf6 entity.name.type.mf6
>  gwf6 sdfdsf#modeltype      namefile      modelname
#^^ source.mf6
#  ^^^^ source.mf6 entity.name.function.mf6
#      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.mf6
#                                           ^^^^^^^^^ source.mf6 entity.name.function.mf6
>  gwf6 sdfdsf #modeltype      namefile      modelname
#^^ source.mf6
#  ^^^^ source.mf6 entity.name.function.mf6
#      ^^^^^^^^ source.mf6
#              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.mf6 comment.mf6
>         GWF6     model.nam          GWF_1
#^^^^^^^^^ source.mf6
#         ^^^^ source.mf6 entity.name.function.mf6
#             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.mf6
>END MODELS
#^^^ source.mf6 keyword.control.mf6
#   ^ source.mf6
#    ^^^^^^ source.mf6 entity.name.type.mf6
>
>  BEGIN     EXCHAngES
#^^ source.mf6
#  ^^^^^ source.mf6 keyword.control.mf6
#       ^^^^^ source.mf6
#            ^^^^^^^^^ source.mf6 entity.name.type.mf6
>  BEGIN     EXCHAngES.
#^^ source.mf6
#  ^^^^^ source.mf6 keyword.control.mf6
#       ^^^^^^^^^^^^^^^^ source.mf6
>  BEGIN     .EXCHAngES
#^^ source.mf6
#  ^^^^^ source.mf6 keyword.control.mf6
#       ^^^^^^^^^^^^^^^^ source.mf6
>  BEGIN.     EXCHAngES
#^^^^^^^^^^^^^ source.mf6
#             ^^^^^^^^^ source.mf6 entity.name.function.mf6
>  .BEGIN     EXCHAngES
#^^^^^^^^^^^^^ source.mf6
#             ^^^^^^^^^ source.mf6 entity.name.type.mf6
>END EXCHANGES
#^^^ source.mf6 keyword.control.mf6
#   ^ source.mf6
#    ^^^^^^^^^ source.mf6 entity.name.type.mf6
>
>BEGIN
#^^^^^ source.mf6 keyword.control.mf6
>  BEGIN
#^^ source.mf6
#  ^^^^^ source.mf6 keyword.control.mf6
>  .BEGIN.
#^^^^^^^^^^ source.mf6
>  BEGIN.
#^^^^^^^^^ source.mf6
>  .BEGIN
#^^^^^^^^^ source.mf6
>    dfdf CONSTANT #dfdf  dfdfdf
#^^^^^^^^^ source.mf6
#         ^^^^^^^^ source.mf6 entity.other.attribute-name.mf6
#                 ^ source.mf6
#                  ^^^^^^^^^^^^^ source.mf6 comment.mf6
>    dfdf .CONSTANT
#^^^^^^^^^^^^^^^^^^^ source.mf6
>     ...OPEN/CLOSE
#^^^^^^^^^^^^^^^^^^^ source.mf6
>     OPEN/CLOSE
#^^^^^ source.mf6
#     ^^^^^^^^^^ source.mf6 entity.other.attribute-name.mf6
>     internal   factor #df
#^^^^^^^^^^^^^^^^ source.mf6
#                ^^^^^^ source.mf6 entity.other.attribute-name.mf6
#                      ^ source.mf6
#                       ^^^ source.mf6 comment.mf6
>     Internal  factor#dfdf
#^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.mf6
>        .gwf6-gwf6.
#^^^^^^^^^^^^^^^^^^^^ source.mf6
>        gwf6-gwt6
#^^^^^^^^ source.mf6
#        ^^^^^^^^^ source.mf6 entity.name.function.mf6
>        gwf6 - gwf6
#^^^^^^^^ source.mf6
#        ^^^^ source.mf6 entity.name.function.mf6
#            ^^^ source.mf6
#               ^^^^ source.mf6 entity.name.function.mf6
>
>    (BINARY)   12323
#^^^^ source.mf6
#    ^^^^^^^^ source.mf6 entity.other.attribute-name.mf6
#            ^^^^^^^^^ source.mf6
>    .(BINARY)   12323
#^^^^^^^^^^^^^^^^^^^^^^ source.mf6
>    (BINARY).   12323
#^^^^^^^^^^^^^^^^^^^^^^ source.mf6
>
>alh
#^^^ source.mf6 entity.name.function.mf6
>.alh.
#^^^^^^ source.mf6
>alh.
#^^^^^ source.mf6
>.alh
#^^^^^ source.mf6
>
>
>drop
#^^^^ source.mf6 entity.name.function.mf6
>.drop.
#^^^^^^^ source.mf6
>drop.
#^^^^^^ source.mf6
>drop.
#^^^^^^ source.mf6
>